src/auditor/auditor.py

```diff
@@ -213,19 +231,19 @@ def check_items(self, report_dir=None):
                 self.report_dict[itemid] = {}
 
                 checker = checks.ItemChecker(item, self.metatable_dict)
-                retry(lambda: checker.setup(credentials.DB))
+                retry(lambda: checker.setup(credentials.DB))  # pylint: disable=W0640
```

Comment:
**suggestion**: i would suggest to be consistent with the disable types. I prefer the spelled out version to remove the need to go look up w0640.